server/src/interfaces/event.interface.ts

```diff
@@ -14,7 +14,7 @@ type EmitEventMap = {
   'config.validate': [{ newConfig: SystemConfig; oldConfig: SystemConfig }];
 
   // album events
-  'album.update': [{ id: string; updatedBy: string }];
+  'album.update': [{ id: string; deliverTo: string[] }];
```

Comment:
We have been using the recipient term, so maybe here we can use `receipientIds`

Reply:
Renamed to `recipientIds`.